debian/rules

```diff
@@ -22,9 +23,15 @@ override_dh_auto_configure:
 	# Also remove usr/include/KHR/khrplatform.h and
 	# usr/lib/pkgconfig/gl.pc. We are using Debian's
 	# default ones which comes from libgl-dev package.
-	find . -iname khrplatform.h -exec rm -vf '{}' \;
+	rm -rvf `find . -iname khr`
```

Comment:
Why the backticks instead of `$()`?
Not asking to change, just making sure you understand the gotchas when using backticks.

Reply:
basically, copy and pasta from another d/rules. As far as I understand, the `$()` is the [new command substitution syntax and the benefit is it doesn't treat anything specially](https://www.gnu.org/software/bash/manual/html_node/Command-Substitution.html), while the old-style backticks treat backticks specially and also backslashes preceding `$`, `\` and \` (interestingly I'm not finding a way to put the backtick on code block style :-). If we were to use `$()`, we would need:

```
rm -rvf $$(find . -iname khr)
```

To protect the `$` which I find a bit confusing at first sight. Anyway, if you prefer it just ask and I can change. Also, if I missed something on the difference between them, let me know.

Reply:
Yea no problems, you got it.